src/Classes/DBPopulator.php

```diff
@@ -0,0 +1,81 @@
+<?php
+namespace ListingsApp\Classes;
+class DBPopulator {
+    private object $listingAPI;
```

Comment:
This can be type hinted as the actual class name ListingAPI rather than just object